### PR TITLE
[EN] Yohane

### PR DESCRIPTION
**Pull request by Konacchi**

### CODE REVIEW

File: en/yohane.md [hunk 1]
```diff
@@ -2,33 +2,33 @@
...
-- `!help` - Memberikan daftar bantuan
-- `!subscribe pp [options]` - Untuk menampilkan status pp seperti limit dan notifikasi jika sudah selesai [submit score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options : `set`,`get`,`unset`,`toggle`)
-- `!boardshow [options]` - [Experimental] Untuk menyembunyikan diri dari leaderboard (options : `others`,`normal`)
-- `!report` - Jika kalian menemukan cheater, silahkan gunakan command ini.
-- `!faq list` - Daftar hal yang sering dibutuhkan dalam chat.
+- `!roll [number]` - Returns random number
+- `!help` - Shows a list of all commands
+- `!subscribe pp [options]` - Notifies about the pp status such as limit after [submitting a score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options: `set`,`get`,`unset`,`toggle`)
+- `!boardshow [options]` - [Experimental] Hide yourself from the leaderboard (options: `others`,`normal`)
+- `!report` - If you find a cheater, please use this command.
```
Comment (troke12) [inline en/yohane.md:14]:
```suggestion
- `!report` - If you found a cheater or annoying players, use this command for report to the staff.
```

File: en/yohane.md [hunk 2]
```diff
@@ -46,29 +46,29 @@ Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<User
...
+If you want to use commands through Yohane's private message, you don't need to use the "!"-prefix.
+- Outside of that, you need to use the "!"-prefix to invoke Yohane's commands.
 
 ## YohaneBot Discord
-- `@faq help` - Untuk menampilkan seluruh command
-- `@stats [options] [nickname]` - Untuk menampilkan status profile (options : `vanilla`,`relax`)
-- `@recent [options] [nickname]` - Untuk menampilkan recent score (options : `vanilla`,`relax`)
-- `@top [options] [nickname]` - Untuk menampilkan top score (options : `vanilla`,`relax`)
+- `@faq help` - Shows all commands
+- `@stats [options] [nickname]` - Displays profile status (options: `vanilla`,`relax`)
```
Comment (troke12) [inline en/yohane.md:72]:
```suggestion
- `@stats [options] [mode]` - Display profile status (options: `vanilla`, `relax`) (mode: `std`, `taiko`, `mania`, `ctb`)

File: en/yohane.md [hunk 3]
```diff
@@ -46,29 +46,29 @@ Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<User
...
+- `@recent [options] [nickname]` - Displays the most recent score (options: `vanilla`,`relax`)
```
Comment (troke12) [inline en/yohane.md:73]:
```suggestion
- `@recent [options]` - Displays the most recent score (options: `vanilla`, `relax`)
```

File: en/yohane.md [hunk 4]
```diff
@@ -46,29 +46,29 @@ Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<User
...
+- `@top [options] [nickname]` - Displays the top score (options: `vanilla`,`relax`)
```
Comment (troke12) [inline en/yohane.md:74]:
```suggestion
- `@top [options]` - Displays the top score (options: `vanilla`, `relax`).
```

File: en/yohane.md [hunk 5]
```diff
@@ -2,33 +2,33 @@
...
 ## Yohane In-Game
 
 ### General commands
-- `!roll [angka]` - Memberikan random number
-- `!help` - Memberikan daftar bantuan
-- `!subscribe pp [options]` - Untuk menampilkan status pp seperti limit dan notifikasi jika sudah selesai [submit score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options : `set`,`get`,`unset`,`toggle`)
-- `!boardshow [options]` - [Experimental] Untuk menyembunyikan diri dari leaderboard (options : `others`,`normal`)
-- `!report` - Jika kalian menemukan cheater, silahkan gunakan command ini.
-- `!faq list` - Daftar hal yang sering dibutuhkan dalam chat.
+- `!roll [number]` - Returns random number
```
Comment (XDerbyX) [inline en/yohane.md:10]:
```suggestion
- `!roll [number]` - Roll a random set of number
```

File: en/yohane.md (hunk 1 above)
Comment (XDerbyX) [reply en/yohane.md:14]:
```suggestion
- `!report` - If you encounter any cheater or an annoying player, use this command to report it to the staff.
```

File: en/yohane.md [hunk 6]
```diff
@@ -2,33 +2,33 @@
...
+- `!help` - Shows a list of all commands
+- `!subscribe pp [options]` - Notifies about the pp status such as limit after [submitting a score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options: `set`,`get`,`unset`,`toggle`)
+- `!boardshow [options]` - [Experimental] Hide yourself from the leaderboard (options: `others`,`normal`)
+- `!report` - If you find a cheater, please use this command.
+- `!faq list` - Returns a list of things that are often needed in chat.
 
 ### Multiplayer commands
 
-Semua command yang ditampilkan disini hanya yang secara umum atau membutuhkan penjelasan. Mohon gunakan `!mp help` apabila bingung dengan command yang tidak terdaftar.
+These are general commands for the use in multiplayer rooms. Please use `!mp help` to show more commands.
```
Comment (XDerbyX) [inline en/yohane.md:19]:
```suggestion
Commands that are shown here are for general use. Please use `!mp help` to show more commands.
```
translasi multiplayer nya dari mana?

Comment (troke12) [reply en/yohane.md:19]: > translasi multiplayer nya dari mana?

belum di translate

File: en/yohane.md [hunk 7]
```diff
@@ -2,33 +2,33 @@
...
+- `!mp close` - Closes the current multiplayer room.
+- `!mp checkperm` - Returns the permission level of the sender.
+- `!mp listref` - Displays a list of referees in the room.
+- `!mp addref <username>` - Adds user as a referee in the room.
+- `!mp rmref <username>` - Removes user as referee in the room.
+- `!mp start [timer]` - Sets all players to ready automatically and starts the game after a timeout (Default: `0`). Can be used to play solo in a multiplayer room.
+- `!mp abort` - Aborts the game. *This is different from using the `Escape`-key in solo*
 
-Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<UserID>` sebagai penggantinya.
+Note: For every operation using the username lookup, you can use `#<UserID>` instead.
```
Comment (XDerbyX) [inline en/yohane.md:31]:
```suggestion
Note: For every operation that use Username Lookup, you can use `#<UserID>` instead.
```

File: en/yohane.md [hunk 8]
```diff
@@ -46,29 +46,29 @@ Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<User
 - `!fokabot reconnect` - Reconnect YohaneBOT if he's not on online users list anymore
```
Comment (XDerbyX) [inline en/yohane.md:46]: Yoooo BOT nya laki? he???

Comment (troke12) [reply en/yohane.md:46]: change to `!bot reconnect` also `"she"` because that is from ripple bot

File: en/yohane.md [hunk 9]
```diff
@@ -46,29 +46,29 @@ Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<User
...
+    | `command`\* | `auto-rank`, `auto-love`, `auto-cancel` | **Can only be set by someone permitted to manage autoranks**<br>Sets the eligibility of a map to get auto-rank. |
+    | `type` | `set`, `set-of`, `map` | `set` - uses **mapset ID** to retrieve data.<br>`set-of` - uses **map ID** to get **a list of maps with the mapset ID** from the given **map ID**.<br>`map` - uses **map ID** to retrive data. |
     
-    Untuk lebih jelasnya, dapat memperhatikan [contoh berikut](https://cdn.discordapp.com/attachments/265909019976138754/801471868699410473/unknown.png)
+    For more details, you may take a look at [this example](https://cdn.discordapp.com/attachments/265909019976138754/801471868699410473/unknown.png).
 
 ### Note
-Jika kalian ingin menggunakan command melalui private message Yohane, tidak perlu menggunakan prefiks-"!"
-- Diluar itu, prefiks-"!" diperlukan untuk memanggil command Yohane.
+If you want to use commands through Yohane's private message, you don't need to use the "!"-prefix.
```
Comment (XDerbyX) [inline en/yohane.md:67]:
```suggestion
If you're using commands through Yohane's private message, you can type your command without the "!"-prefix.
```

File: en/yohane.md (hunk 6 above)
Comment (XDerbyX) [reply en/yohane.md:19]: oh woop
s gw salah liat tadi

File: en/yohane.md (hunk 1 above)
Comment (XDerbyX) [reply en/yohane.md:14]: atau gini
```suggestion
- `!report` - If you encounter any cheater or an annoying player, use this command to notify our staff.
```

File: en/yohane.md [hunk 10]
```diff
@@ -2,33 +2,33 @@
...
-- `!mp checkperm` - Menampilkan permission level pada pengirim.
-- `!mp listref` - Menampilkan referee pada room.
-- `!mp addref <username>` - Menambahkan user menjadi referee pada room.
-- `!mp rmref <username>` - Mengeluarkan user dari referee pada room.
-- `!mp start [timer]` - Membuat semua pemain siap secara otomatis dan memulai permainan setelah waktu habis (Default: `0`). Dapat dilakukan untuk solo-play pada multi room.
-- `!mp abort` - Untuk membatalkan permainan tersebut. *Hal ini berbeda dengan menggunakan `Escape`-key pada solo-play*
+- `!mp help` - Shows a list of all multiplayer commands.
+- `!mp make [name]` - Creates a multiplayer room.
+- `!mp close` - Closes the current multiplayer room.
+- `!mp checkperm` - Returns the permission level of the sender.
```
Comment (XDerbyX) [inline en/yohane.md:24]:
```suggestion
- `!mp checkperm` - Display your permission level for the current room. 
```

File: en/yohane.md [hunk 11]
```diff
@@ -2,33 +2,33 @@
...
+- `!mp listref` - Displays a list of referees in the room.
+- `!mp addref <username>` - Adds user as a referee in the room.
+- `!mp rmref <username>` - Removes user as referee in the room.
+- `!mp start [timer]` - Sets all players to ready automatically and starts the game after a timeout (Default: `0`). Can be used to play solo in a multiplayer room.
```
Comment (XDerbyX) [inline en/yohane.md:28]:
```suggestion
- `!mp start [timer]` - Automatically sets all players to be ready and starts the game after a countdown (Default: `0`). Can be used to play solo in a multiplayer room.
```

File: en/yohane.md (hunk 1 above)
Comment (Konacchi) [reply en/yohane.md:14]: You wouldn't add "annoying" in the sense of a report in a English doc, as it is too subjective

File: en/yohane.md (hunk 5 above)
Comment (Konacchi) [reply en/yohane.md:10]: "set of number" is grammatically incorrect, and as it only returns one number, referring to it as "set" would be wrong

Comment (Konacchi) [reply en/yohane.md:10]: I would also suggest to change the argument from "number" with eg "max" (I think it's the maximum amount? It's not really clear based on the given description)

File: en/yohane.md (hunk 6 above)
Comment (Konacchi) [reply en/yohane.md:19]: Very "wonky" formulation. Will rephrase it for the sake of correct English. Also, please keep comments in English.

Comment (Konacchi) [reply en/yohane.md:19]: Actually, the description is kinda redundant with the title and the list itself (as we mention the help twice in a row), we can just leave that out entirely imo.

File: en/yohane.md (hunk 7 above)
Comment (Konacchi) [reply en/yohane.md:31]: This is not English

File: en/yohane.md (hunk 8 above)
Comment (Konacchi) [reply en/yohane.md:46]: I will also take the opportunity to correct the sentence

File: en/yohane.md (hunk 9 above)
Comment (Konacchi) [reply en/yohane.md:67]: word doubling with "command", I'd only change the first part

File: en/yohane.md (hunk 7 above)
Comment (XDerbyX) [reply en/yohane.md]: Does it need a quotation mark for the command Username Lookup? like
```suggestion
Note: For every operation that uses 'Username Lookup' can use `#<UserID>` instead.

File: en/yohane.md (hunk 9 above)
Comment (XDerbyX) [reply en/yohane.md]: could remove the latter

File: en/yohane.md (hunk 5 above)
Comment (XDerbyX) [reply en/yohane.md]: Kind of confused on how to word it. I will leave it up to you 🤔

File: en/yohane.md (hunk 1 above)
Comment (XDerbyX) [reply en/yohane.md]: Ah, that's fair, any suggestion on what word to use?
Maybe misbehaving?

File: en/yohane.md (hunk 6 above)
Comment (XDerbyX) [reply en/yohane.md]: We could, since the title itself in self-explanatory

File: en/yohane.md (hunk 5 above)
Comment (Konacchi) [reply en/yohane.md]: I already did updates on the last commit, feel free to check

File: en/yohane.md (hunk 1 above)
Comment (Konacchi) [reply en/yohane.md]: Well, we could describe it generally as people who break the rules or something like that

File: en/yohane.md (hunk 6 above)
Comment (Konacchi) [reply en/yohane.md]: Alright, will cut it in the next commit.

File: en/yohane.md (hunk 7 above)
Comment (Konacchi) [reply en/yohane.md]: We can do that, but I wouldn't say it's necessary. Whatever you prefer.

File: en/yohane.md (hunk 1 above)
Comment (XDerbyX) [reply en/yohane.md]: Ah, good idea. Maybe something like this, up to you.
```suggestion
- `!report` - If you encounter any player breaking rules, use this command to notify our staff.
```

File: en/yohane.md (hunk 5 above)
Comment (XDerbyX) [reply en/yohane.md]: Checked, it's good 😎

File: en/yohane.md (hunk 7 above)
Comment (XDerbyX) [reply en/yohane.md]: Ah, the original file in ID doesn't have quotation, but it does capitalize the commands, so maybe just this.
```suggestion
Note: For every operation using the Username Lookup, you can use `#<UserID>` instead.
```

Comment (Konacchi) [reply en/yohane.md]: it is already changed in the latest commit

Comment (XDerbyX) [reply en/yohane.md]: Oh sorry, nevermind then

**Comment by Konacchi** [comment]
@XDerbyX I think we catched everything pointed out before now. Please check the latest commit ~

**Comment by XDerbyX** [comment]
@Konacchi Yep! Checked and all seems fine to me. 😁

**Comment by troke12** [comment]
alright, thank you @Konacchi and @XDerbyX !
i will merge this

**Pull request merged**